src/Microsoft.Data.Analysis/Microsoft.Data.Analysis.csproj

```diff
@@ -18,6 +18,16 @@
       <AutoGen>True</AutoGen>
       <DependentUpon>Converters.tt</DependentUpon>
     </None>
+    <None Include="DataFrameColumn.BinaryOperationImplementations.Exploded.cs">
```

Comment:
Is there a reason these files are called `DataFrameColumn.XXX` but really only apply to `PrimitiveDataFrameColumn`s?

Reply:
I think my reasoning was that these are now exploded column types and not `PrimitiveDataFrameColumn`s. I don't have a strong preference here, I can rename the files.

Reply:
We should be consistent. The operators: `+`, `-`, etc. are in `PrimitiveDataFrameColumn.BinaryOperators`. But these aren't.

IMO, it would make the most sense for them all to be in files named `PrimitiveDataFrameColumn.XXX`, since they only operate on primitive columns.

Reply:
Ok, renamed everything to PrimitiveDataFrameColumn.XXX